.npmrc

```diff
@@ -0,0 +1 @@
+hoist=false
```

Comment:
nit: maybe add a comment for what this is for

Reply:
👍